src/VisualStudio/Core/Def/Implementation/Remote/JsonRpcConnection.cs

```diff
@@ -64,6 +65,16 @@ internal sealed class JsonRpcConnection : RemoteServiceConnection
             }
         }
 #endif
+
+        internal void SetPoolReclamation(IPooledConnectionReclamation poolReclamation)
+        {
+            _ = poolReclamation ?? throw new ArgumentNullException(nameof(poolReclamation));
```

Comment:
`Contract.NotNull()` as this is not public API.

Reply:
updated